tex/hw03/hw03.tex

```diff
@@ -0,0 +1,70 @@
+\documentclass{scrartcl}
+\usepackage{tikz}
+\usepackage{amsmath}
+\usepackage{amssymb}
+\usepackage{unicode-math}
+\setmathfont{XITS Math}
+\usepackage{forest}
+
+\newcommand{\selection}{\sigma}
+\newcommand{\projection}{\pi}
+\newcommand{\rename}{\rho}
+\newcommand{\join}{\Join}
+% \fullouterjoin defined by unicode-math
+\newcommand{\semijoin}{\ltimes}
+\newcommand{\groupby}{\Gamma}
+
+\newcommand{\mtt}[1]{\text{\texttt{#1}}}
+
+\setlength{\parindent}{0pt}
+
+\begin{document}
+
+\section*{Exercise 1}
+
+\subsection*{Bad Selection Estimation}
+Assume we have a table t1 with 1100 elements, where 1000 elements are with the value x = 0. The other 100 elements get
+the numbers $x = \{1\dots100\}$, s.t.\ each x value of these remaining elements is unique. Furthermore, we have a second
+attribute y with the value's $y = \{1\dots50\}$, s.t.\ each y value appears the same amount of times in the table t1.
+
+Assuming our non key fromula $$\frac{1}{|t1.x|}$$ we get selectivity of $\frac{1}{101}$ for x and $\frac{1}{50}$ for y.
```

Comment:
come on, you'd be faster to just fix the typo in tex directly and push the update ^^